src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthModule.TransactionExecutor.cs

```diff
@@ -63,6 +67,24 @@ protected TxExecutor(IBlockchainBridge blockchainBridge, IBlockFinder blockFinde
 
                 FixCallTx(transactionCall);
 
+                if ((transactionCall.MaxFeePerGas != null || transactionCall.MaxPriorityFeePerGas != null) &&
+                    transactionCall.GasPrice != null)
+                {
+                    return ResultWrapper<TResult>.Fail("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified", ErrorCodes.InvalidInput);
```

Comment:
It could be done in the next PRs, but please remember about other RPC methods like eth_sendTransaction (to verify in geth)